### PR TITLE
get_complete_position supports imports and panic handling

### LINKED ISSUE #46: Does not support 'github.c...' in import completion
Go built-in package is working perfectly.
but, third-party package (e.g. `github.com/tools/godep`, `golang.org/x/tools`).
Caused by dot (`.`) in the domain name.

Ref: 
- https://github.com/zchee/deoplete-go/issues/32#issuecomment-195541918
- https://asciinema.org/a/3xz29k70w2zmi27sg2zugde74

### PR DESCRIPTION
**Pull request by nhooyr**
This closes #46 and #45

### CODE REVIEW

**Comment by nhooyr** [comment]
Actually don't merge, I think I can make it a bit more clean.

**Comment by zchee** [comment]
@nhooyr OK :)

**Comment by nhooyr** [comment]
@zchee you might know how to do this.

If we get the string `"github.com/` with `re.search` I want to match `"github.com` but make `re.start()` act like the pattern started at `g`. So if I have the following:

`"github.com|`

I want the regex to match the entire string with the `"` but make re.start() return the position at g. 

`"|github.com` not `|"github.com`.

Obviously this shouldn't happen for patterns like `fmt.Println`. So I have two regexes, one that matches the import statement, other that matches normal statements. Instead I can do it in a single statement if python regexes supports this.

**Comment by nhooyr** [comment]
actually never mind, its fine.

File: rplugin/python3/deoplete/sources/go.py [hunk 1]
```diff
@@ -34,8 +34,12 @@ def __init__(self, vim):
         self.sort_class = self.vim.vars['deoplete#sources#go#sort_class']
 
     def get_complete_position(self, context):
-        m = re.search(r'\w*$', context['input'])
-        return m.start() if m else -1
+        m = re.search(r'"[^"]*$', context['input'])
```
Comment (zchee) [inline rplugin/python3/deoplete/sources/go.py]: Maybe, this regex is better than it. (slightly tested)

``` python
m = re.search(r'(\w*$|(?<=")([\.\/\w]*$))', context['input'])
```

Not required `if...else` if use this regex.
Could you test it?

Comment (nhooyr) [inline rplugin/python3/deoplete/sources/go.py]: yup, thats what I was asking about, in my big post. how to do that in regex.

**Comment by nhooyr** [comment]
@zchee That regex you commented is exactly what I was asking about. I just didn't know the syntax in python. I cleaned it up a bit and updated the PR.

**Comment by zchee** [comment]
@nhooyr LGTM. Thanks!!

**Pull request merged**